.github/ISSUE_TEMPLATE/compatible-certification-request.md

```diff
@@ -0,0 +1,18 @@
+- [ ] Organization Name ("Organization") and, if applicable, URL:<br/>
```

Comment:
You should include the metadata header to make this template more useful, just like the spec_issue.md template:
```
---
name: Compatible Certification Request
about: Start a request for a compatible certification 
title: ''
label: Certification
assignees: ''
---
```